### PR TITLE
[spec][wercker] Remove Django build require from hatohol package in spec

### PR DESCRIPTION
**Pull request by cosmo0920**
It got success building Hatohol rpms in Wercker: https://app.wercker.com/#buildstep/569c71813e708cf423045b51

### CODE REVIEW

File: hatohol.spec.in [hunk 1]
```diff
@@ -32,7 +32,6 @@ BuildRequires: mariadb-devel
 %endif
 BuildRequires: libuuid-devel >= 2.17
 BuildRequires: librabbitmq-devel >= 0.4.1
-BuildRequires: Django >= 1.5.3
```
Comment (Mnakagawa) [inline hatohol.spec.in]: should we add the following line?
```
BuildRequires: gettext
```

Comment (cosmo0920) [inline hatohol.spec.in]: Yes, we should. Good catch! :+1:

**Comment by Mnakagawa** [comment]
Looks so good:+1:
Let us wait to finish the test.

**Comment by cosmo0920** [comment]
Yesy, we should wait finishing test... :sleepy:

**Comment by cosmo0920** [comment]
Thanks!

**Pull request merged**